packages/atlas/src/providers/assets/assets.provider.tsx

```diff
@@ -145,6 +151,39 @@ export const OperatorsContextProvider: FC<PropsWithChildren> = ({ children }) =>
     fetchStorageOperators()
   }, [fetchStorageOperators, isStudio])
 
+  useMountEffect(() => {
+    const initBenchmark = async () => {
+      const { data } = await getBasicVideo({
+        variables: {
+          limit: 1,
+          where: {
+            thumbnailPhoto: {
+              isAccepted_eq: true,
+            },
+          },
+        },
+      })
+      const thumbnail = data?.videos[0].thumbnailPhoto
+      if (thumbnail) {
+        const promiseRace = getFastestImageUrl(thumbnail.resolvedUrls)
+        const startTime = performance.now()
+        await promiseRace
+        const msDuration = performance.now() - startTime
+        if (
+          msDuration > atlasConfig.storage.assetResponseTimeout ||
+          msDuration < atlasConfig.storage.assetResponseTimeout / 2
+        ) {
+          setUserBenchmarkTime(msDuration * 1.5)
```

Comment:
this won't be overwritten once set, as it always comparing it to the config const. I think it should be a variable with the val = `userBenchmarkTime || atlasConfig.storage.assetResponseTimeout`

Reply:
also, to exclude time spikes (in both ways) I think it might be good to take and average value between current `userBenchmarkTime || atlasConfig.storage.assetResponseTimeout` and new `msDuration`.  So It'll look smth like this: `setUserBenchmarkTime((userBenchmarkTime || atlasConfig.storage.assetResponseTimeout + msDuration) * 0.75) // / 2 * 1.5 === * 0.75`